presto-elasticsearch/src/main/java/com/facebook/presto/elasticsearch/ElasticsearchClient.java

```diff
@@ -70,20 +70,19 @@
     private static final Logger LOG = Logger.get(ElasticsearchClient.class);
```

Comment:
please squash it, this was just an idea (POC), not production ready code. Surely it requires more work.